doc/rados/configuration/mon-config-ref.rst

```diff
@@ -600,6 +600,88 @@ is far outweighed by the number of accidental pool (and thus data) deletions it
 
 For more information about the pool flags see :ref:`Pool values <setpoolvalues>`.
 
+Monitor backup
+==============
+
+The monitor daemon is capable of creating consistant backups of the RocksDB to allow restorating
+in case of fatal multi node failure. It uses the native RocksDB backup functionality to create consistant 
+snapshots that can be backuped elsewhere without downtime.
+
+When :ref:`Backup Interval<mon_backup_interval>` is set, the backup is triggered the specified interval in minutes.
+
+When :ref:`Auto Backup <mon_auto_backup>` is enabled, creation of certain objects like auth
+permissions trigger a backup.
+
+When :ref:`Auto Backup <mon_auto_backup>` is enabled, creation of certain objects like auth
+permissions trigger a backup.
+
+To reduce IO on mon database, shared backups are the default, but every 5th backup or manually triggered backups are full backups.
```

Comment:
In order to limit demands on the Ceph Monitor database, shared backups are the default.  Every fifth backup, or any manually-triggered backup however will be a full backup.